travis.sh

```diff
@@ -19,6 +19,9 @@ set -x
 set -o pipefail
 shopt -s globstar
 
+add_ppa 'ppa:cwchien/gradle'
+sudo apt-get -qqy install gradle-3.0
```

Comment:
Travis might not like this. Probably have to add this stuff to the `.travis.yml` instead https://docs.travis-ci.com/user/installing-dependencies/#Installing-Packages-with-the-APT-Addon (or change to VM-based builder with `sudo: required` in `.travis.yml` https://docs.travis-ci.com/user/ci-environment/#Virtualization-environments.